actors/user/src/test/java/org/sunbird/user/EducationManagementActorTest.java

```diff
@@ -0,0 +1,251 @@
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.Request;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.user.actors.EducationManagementActor;
+import org.sunbird.user.dao.EducationDao;
+import org.sunbird.user.dao.impl.EducationDaoImpl;
+import org.sunbird.user.util.UserActorOperations;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ServiceFactory.class, EducationDaoImpl.class})
+@PowerMockIgnore({"javax.management.*"})
+public class EducationManagementActorTest {
+
+  private static final ActorSystem system = ActorSystem.create("system");
+  private static final Props props = Props.create(EducationManagementActor.class);
+  private static final CassandraOperation cassandraOperation = mock(CassandraOperationImpl.class);
+
+  @Before
+  public void beforetest() {
+    PowerMockito.mockStatic(EducationDaoImpl.class);
+    EducationDao educationDao = Mockito.mock(EducationDaoImpl.class);
+    when(EducationDaoImpl.getInstance()).thenReturn(educationDao);
+    when(educationDao.getPropertiesValueById(Mockito.anyString(), Mockito.anyString()))
+        .thenReturn(getEducationResponse());
+  }
+
+  @BeforeClass
+  public static void beforeClass() {
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    when(cassandraOperation.insertRecord(
+            Mockito.anyString(), Mockito.anyString(), Mockito.anyMap()))
+        .thenReturn(getSuccessResponse());
+    when(cassandraOperation.deleteRecord(
+            Mockito.anyString(), Mockito.anyString(), Mockito.anyString()))
+        .thenReturn(getSuccessResponse());
+  }
+
+  private static Response getEducationResponse() {
+    Response response = new Response();
+    List<Map<String, Object>> lst = new ArrayList<>();
+    Map<String, Object> map = new HashMap<>();
+    map.put(JsonKey.ADDRESS_ID, "someAddressId");
+    lst.add(map);
+    response.put(JsonKey.RESPONSE, lst);
+    return response;
+  }
+
+  @Test
+  public void testInsertEducationSuccess() {
+
+    boolean result =
+        testScenario(
+            UserActorOperations.INSERT_USER_EDUCATION,
+            true,
+            true,
+            getUpsertResponse(),
+            true,
+            true,
+            true);
+    assertTrue(result);
+  }
+
+  @Test
+  public void testUpdateEducationForDeleteEducationDetailsSuccess() {
+
+    boolean result =
+        testScenario(
+            UserActorOperations.UPDATE_USER_EDUCATION,
+            true,
+            true,
+            getUpsertResponse(),
+            true,
+            true,
+            true);
+    assertTrue(result);
+  }
+
+  @Test
+  public void testUpdateEducationForDeleteEducationDetailsWithoutAddressSuccess() {
```

Comment:
testUpdateEducationSuccessForDeleteEducationDetailsWithoutAddress

But how does it matter? delete would require only id i guess